app/forms/reminder_send_form.rb

```diff
@@ -0,0 +1,22 @@
+class ReminderSendForm
+  include ActiveModel::Model
+
+  attr_accessor(
+    :email,
+  )
+
+  validates :email, presence: true, format: { with: /\A\S+@.+\.\S+\z/ }
+  validate :user_valid?
+
+  def user
+    User.find_by(email: email)
+  end
+
+  private
+
+  def user_valid?
+    if user.blank?
+      errors.add(:email, 'email not found!')
```

Comment:
The name of the method implies that the only thing it should do is returning `true` or `false`. But here you have a side effect of adding errors to the form. If you want to add errors here, name the method differently.

https://ru.wikipedia.org/wiki/CQRS
https://ru.hexlet.io/courses/js-functions/lessons/command-query-separation/theory_unit (if you have a subscription)